x-pack/plugins/session_view/public/hooks/use_process_tree.ts

```diff
@@ -164,6 +164,12 @@ class ProcessImpl implements Process {
       ({ event }) => event.action === EventAction.exec || event.action === EventAction.fork
     );
 
+    const execs = execsForks.filter(({ event }) => event.action === EventAction.exec);
```

Comment:
Maybe `reduce` `this.events.filter` to separate events and forks so we don't need to iterate over some of the events twice?